sui-execution/Cargo.toml

```diff
@@ -31,6 +31,8 @@ sui-verifier-v1 = { path = "v1/sui-verifier" }
 sui-verifier-v2 = { path = "v2/sui-verifier" }
 # sui-verifier-$CUT = { path = "$CUT/sui-verifier" }
 
+move-abstract-interpreter-latest = { path = "../external-crates/move/crates/move-abstract-interpreter", package = "move-abstract-interpreter" }
```

Comment:
I think you need to put a `-$CUT` template here for `move-abstract-interpreter`.

Reply:
Good catch.. A bit strange that nothing broke without it. Maybe just that the APIs don't change in the test?

Reply:
I suppose if we have no uses of the abstract interpreter from Sui, it won't complain? That might imply that you don't need to add it here at all (until it gets used from the Sui side).